cmd/prometheus/main.go

```diff
@@ -128,12 +128,14 @@ func Main() int {
 	signal.Notify(hup, syscall.SIGHUP)
 	go func() {
 		<-hupReady
+		reloadCh, errCh := webHandler.Reload()
 		for {
 			select {
 			case <-hup:
-			case <-webHandler.Reload():
+				reloadConfig(cfg.configFile, reloadables...)
```

Comment:
I'm not sure what I can do with the error here, because there is no client to send the error, since it's a signal.